cypress/integration/transforms_spec.js

```diff
@@ -0,0 +1,247 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+import { PLUGIN_NAME } from "../support/constants";
+import sampleTransform from "../fixtures/sample_transform";
+
+const TRANSFORM_ID = "test_transform_id";
+
+describe("Transforms", () => {
+    beforeEach(() => {
+      cy.deleteAllIndices();
```

Comment:
Do you want to delete and load the sample data for every test? That should be unchanging I would imagine?